bitnami/concourse/values.yaml

```diff
@@ -67,7 +67,7 @@ diagnosticMode:
 image:
```

Comment:
Please remove this change from the PR since it belongs to concourse

Reply:
Done.Please review it again.Thank you^_^